lib/solidus_social/config.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'solidus_social/social_configuration'
+
+module Spree
+  def self.social_config
+    @config ||= Spree::SocialConfiguration.new
+  end
+
+  def self.const_missing(name)
+    name == :SocialConfig ? social_config : super
```

Comment:
Wouldn't it work to append

```ruby
::Spree::SocialConfig = Spree::SocialConfiguration.new
```

in `lib/solidus_social/social_configuration.rb`?

Reply:
Modified this here [4b71338](https://github.com/solidusio-contrib/solidus_social/commit/4b71338044bcb42b46f8e8db2064cf33bae63920)